### PR TITLE
Issue #48: New Community page - initial content

### PR DESCRIPTION
**Pull request by brianking**
Signed-off-by: Brian King <brian.king@eclipse-foundation.org>

### CODE REVIEW

File: content/community/_index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+title: "Community"
+headline: "Get Involved"
+tagline: ECD Tools is a community of members, adopters, developers, and other contributors. Join us!
+date: 2020-09-30T14:53:00+02:00
+hide_page_title: true
+hide_sidebar: true
+show_featured_footer: false
+container: "container-fluid"
+---
+{{< grid/section-container>}}
```
Comment (chrisguindon) [inline content/community/_index.md:11]: You can remove these 2 lines here, the two closing tags at the bottom and line 9: container: "container-fluid".

**Review by chrisguindon — CHANGES_REQUESTED** [review]
The next question is how do people get access to this page?

File: content/community/_index.md (hunk 1 above)
Comment (brianking) [reply content/community/_index.md:11]: But then that would remove the top jumbotron, right?

**Comment by brianking** [comment]
> The next question is how do people get access to this page?

I changed config.toml to add a top menu entry for it, have idea why it was not included in the PR.

**Pull request closed without merging**